README.md

```diff
@@ -1,342 +1,37 @@
-# The Hurricane Florence API
+# The Hurricane Response API
 
-* We serve JSON data here, open and fresh
-* We help client applications help those affected by Hurricane Florence.
+This API Server is the backend data management dashboard for mapping community resources such as Shelter and Food Distribution Points in disaster-effected regions.  It has importers for data from a FEMA managed GeoServer that contains data supplied by the RedCross, and user interfaces for manually updating.
+
+There are planned features to have crowdsourced update tools to help the community update the data in real time as shelter/distribution point records become stale.
+
+## History
+
+The Hurricane API was born out of necessity during Hurricane Harvey in 2017 when Houston, TX was innundated with a storm and people struggled to location shelters.  The developers at Sketch City took it upon themselves to crowdsource a solution.  This software is the product of their efforts and has continued to be developed by other Code for America brigade members.
+
+## Objective
+
+* We serve GeoJSON to mapping front-ends
+* We help client applications help those affected by hurricanes and other natural disasters.
 
 ## Developer Quick Links
 
 * [CONTRIBUTORS](https://github.com/hurricane-response/florence-api/graphs/contributors)
 * [GETTING STARTED](#getting-started)
```

Comment:
Heads up that this link is dead (and has been for a while). There's a Getting Started section in the CONTRIBUTING that maybe this could link too? As someone starting out with the code, I was thought GETTING STARTED was most clearly what I wanted to click on, so might be nice to keep it even if it's basically the same link as CONTRIBUTING?

Reply:
Heard you loud and clear.  How does merging the two link items into `[GETTING STARTED/CONTRIBUTING]` pointing to `CONTRIBUTING.md` feel?  I pushed a commit for that, but I can revert if it hit the uncanny valley.